assets/css/admin-notices.scss

```diff
@@ -11,7 +11,7 @@ $notice-success: #43af99;
 
 	// Spacing.
 	gap: 10px;
-	padding: 10px 12px;
+	padding: 10px 12px !important;
```

Comment:
I'm not totally cool with this, but after testing a few other options, it's the most straightforward option and limited to WPJM notice styling.

Another option is to rename `notice` to `wpjm-notice` and copy the relevant styling over to `admin-notices.scss`.

Reply:
I realized an issue that we're also overriding an style that increases the padding in order to not have the dismiss button over the text.

<img width="536" alt="Screenshot 2023-05-05 at 16 28 33" src="https://user-images.githubusercontent.com/876340/236553533-945803be-b69b-48a4-a47a-c08b01b83397.png">

Maybe a solution could be to add the `!important` only on the `padding-top` and `padding-bottom`?

Reply:
Ah, I see what's happening there, yeah. I've gone with the separated `padding` route and I'm not seeing any issues with that.

Committed here https://github.com/Automattic/WP-Job-Manager/pull/2409/commits/b63ae05c8050adb24085c5ed9fdf743fca25abeb